src/js/components/stock-movement-wizard/modals/SubstitutionsModal.jsx

```diff
@@ -87,24 +88,38 @@ const FIELDS = {
               style={{ color: option.color ? option.color : 'black' }}
               className="d-flex align-items-center"
             >
-              {option.label}
-              &nbsp;
-              {renderHandlingIcons(option.value ? option.value.handlingIcons : [])}
+              <Tooltip
```

Comment:
this Product field should probably be replaced by `<ProductSelectFiel />` instead of `<Select />`
I think that when I was working o implementing exact match I might have missed this modal but I think it would be a good idea to replace this with ProductSelectFiel